js/index.js

```diff
@@ -1,6 +1,126 @@
+Aliquam non nisl ultricies, faucibus felis sed, egestas dui. Nam efficitur neque ut quam elementum congue. Praesent arcu diam, egestas non ipsum non, ultricies imperdiet justo. Sed at elementum ligula, et condimentum augue. Phasellus eget urna vel lectus lobortis convallis. Praesent vulputate commodo dui. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin accumsan orci in nisl porttitor, et congue leo efficitur. Curabitur sodales lacus sed placerat porttitor. Quisque nec quam pulvinar, auctor libero nec, porta tellus. Vivamus vitae pellentesque magna. Fusce blandit magna ac justo ornare, quis congue dui aliquet. Vivamus eu arcu urna.
+`
+
+const paragraph2 = "Lorem, ipsum, et dolor, sit, amet, consectetur,  et " 
+
+let countParagraph = paragraph.length
+let numberOfWordsPar1 = 0;
+for (i=0; i<paragraph.length; i++) {
+    if (paragraph[i]===" "){
+        numberOfWordsPar1 = numberOfWordsPar1 + 1 ;
+    }
+}
+console.log(numberOfWordsPar1);
+
+let numberOfEt = 0;
+for (i=0; i<paragraph.length; i++) {
+    if (paragraph[i]+paragraph[i+1]+paragraph[i+2]+paragraph[i+3] === " et ") {
+    numberOfEt = numberOfEt + 1;
+}
+else if (paragraph[i]+paragraph[i+1]+paragraph[i+2]+paragraph[i+3] === " et.") {
+    numberOfEt = numberOfEt + 1;
+}
+}
+console.log(numberOfEt);
+
+/* Question for Karina
+
+for (i=0; i<paragraph2.length; i++) {
+    
+    if (paragraph2.includes(" et ")) {
+        numberOfEt = numberOfEt + 1;
+        console.log("Hi");
+    }
+    else if (paragraph.includes(" et."[i])){
+        numberOfEt = numberOfEt + 1;
+   
+}
+*/
+
+
+// Bonus 3//
+
+let phraseToCheck = "Amor, Roma";
+let phraseToCheckToLowercase = phraseToCheck.toLowerCase();
+let reverseChar = "" ;
+let phraseToCheckToLowercaseNoSpaces ="";
+let reverseCharNoSpaces ="";
+
+
+for (i = phraseToCheck.length - 1; i>=0; i--) {
+    reverseChar += phraseToCheckToLowercase[i];
+}
+
+for (i=0; i<phraseToCheckToLowercase.length; i++) {
+    if (phraseToCheckToLowercase[i] != " " && phraseToCheckToLowercase[i] != ",") {
+        reverseCharNoSpaces += phraseToCheckToLowercase[i];
+    }
+}
+
+for (i=0; i<phraseToCheck.length; i++) {
+    if (phraseToCheckToLowercase[i] != " " && phraseToCheckToLowercase[i] != ",") {
+        phraseToCheckToLowercaseNoSpaces += phraseToCheckToLowercase[i];
+    }
+}
+
+if (phraseToCheckToLowercaseNoSpaces === reverseCharNoSpaces) {
```

Comment:
Bonus 2 isn't working properly. Try checking a phrase that isn't a palindrome

The general idea is correct, you need to remove all special characters and spaces, make everything lower case, after that reverse the order and finally compare i